main.tf

```
@@ -1,17 +1,22 @@
 locals {
-  port                 = var.port == "" ? var.engine == "aurora-postgresql" ? "5432" : "3306" : var.port
+  port                 = var.port == "" ? var.engine == "aurora-postgresql" ? 5432 : 3306 : var.port
```

```suggestion
  port                 = var.port == "" ? (var.engine == "aurora-postgresql" ? 5432 : 3306) : var.port
```

Just to increase readability. I think it helps especially for longer statements.